tests/testthat/test_datasets.R

```diff
@@ -0,0 +1,75 @@
+context("datasets")
+source("utils.R")
+
```

Comment:
also, could you test other remaining functions in e2e to increase our test coverage?